### ISSUE TITLE
[dash] missing syntax highlighter

### ISSUE THREAD

**Issue by galeyang** [flutter/website#1419]
We'd like to ensure ALL code snippets have the syntax highlighter to enhance the usability.

Many pages lose the syntax highlighter. For example, Building Layouts in Flutter, Adding Interactivity to Your Flutter App.

![image](https://user-images.githubusercontent.com/5102864/46699813-cb0b8a00-cbcf-11e8-80d0-33a8d922bb08.png)

**Comment by chalin** [comment]
Yes, this was on my (local) todo list. Thanks for filing an issue about it.
